proto/internal/temporal/server/api/schedule/v1/message.proto

```diff
@@ -155,3 +159,55 @@ message HsmSchedulerState {
     google.protobuf.Timestamp next_invocation_time = 3;
 
 }
+
+// State machine scheduler internal state.
+message SchedulerInternal {
+    temporal.server.api.enums.v1.Scheduler2State state = 1;
+
+    // Scheduler request parameters and metadata. 
+    temporal.api.schedule.v1.Schedule schedule = 2;
+    temporal.api.schedule.v1.ScheduleInfo info = 3;
+    temporal.api.schedule.v1.SchedulePatch initial_patch = 4;
+
+    // State common to all generators is stored in the top-level machine.
+    string namespace = 5;
+    string namespace_id = 6;
+    string schedule_id = 7;
```

Comment:
Note that these docstrings will only be attached to the field in the following line.
You'll want to ensure that all fields have docstrings that directly apply.